components/DanceCritiqueForm/index.js

```diff
@@ -97,7 +97,8 @@ class DanceCritiqueFormInner extends React.Component {
       console.log('yo you\'re missing some required fields');
       // TODO: handle error better
     } else {
-      submitDanceCritique(this.props);
+      this.props.onSubmitDanceCritique(this.props);
```

Comment:
hmmm why is `onSubmitDanceCritique` from `this.props`? And then you're passing in `this.props` into it which will also pass itself into itself

Reply:
ohh okay you have to `mapDispatchToProps` so it's a prop, but then we probably shouldn't pass in all the props (`this.props`) into `this.props.onSubmitDanceCritique`

Reply:
I r changing this nao